templates/santa/application/message_slack.html.twig

```diff
@@ -1,6 +1,22 @@
 {% extends 'santa/message.html.twig' %}
 
 {% block user_item %}
-    <img src="{{ user.extra.image }}" alt="{{ user.name }}" />
+    <img src="{{ user.extra.image }}" alt="{{ user.name }}"/>
     <span title="{{ user.name }} ({{ user.extra.nickname }})">{{ user.name }}</span>
 {% endblock %}
+
+{% block time_limit %}
+    <div class="user-field message-step">
+        <h3>Want to schedule the message to a later date ?</h3>
+
+        <p>If not, you can just leave this field <strong>empty</strong>.
+            <br>
+            The time you will specify here will be interpreted in <strong>UTC</strong>, please mind
+            <strong>timezones</strong> when scheduling your event.
+            <br><br>
+            Ex : If you live in GMT+2, schedule the event for 2 hours sooner than you really want it.
```

Comment:
This is really complicated, we must put our-self in our users shoes. They don't know what a timezone is. They don't care.

To help them you can use JavaScript to convert the time to their local TZ for example.

Reply:
I agree. This was but a temporary solution before we started working on it again.
I will take a look at your JS solution